src/components/Header/Header.tsx

```diff
@@ -75,12 +77,13 @@ const Header = ({
             navbarStatus === true ? styles.showMenu : styles.hideNav
           )}
         >
-          {menuItemData.map((name, index) => {
+          {menuItemData.map((menuItem, index) => {
             return (
               <MenuItem
                 key={`menuItem-${index}`}
```

Comment:
if you have to use index and the key, you can reliably just use `index` instead of creating some key with prefix. the key is scoped to this block of code so you dont have to worry about conflicts